extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/handler/HueBridgeHandler.java

```diff
@@ -274,6 +374,24 @@ private synchronized void onUpdate() {
                 }
                 pollingJob = scheduler.scheduleWithFixedDelay(pollingRunnable, 1, pollingInterval, TimeUnit.SECONDS);
             }
+            if (sensorPollingJob == null || sensorPollingJob.isCancelled()) {
+                int pollingInterval = DEFAULT_SENSOR_POLLING_INTERVAL;
+                try {
+                    Object pollingIntervalConfig = getConfig().get(SENSOR_POLLING_INTERVAL);
+                    if (pollingIntervalConfig != null) {
+                        pollingInterval = ((BigDecimal) pollingIntervalConfig).intValue();
```

Comment:
Are we sure that we can cast here? Does the framework make sure that `getConfig()` always converts the value to a BigDecimal or could it potentially be a String here?

Reply:
The nicest way to make sure we do not run into an exception would be to introduce a dedicated configuration class for it and use the `getConfigAs()` method. I will change it, if possible.